config-example.yaml

```diff
@@ -369,6 +369,11 @@ unix_socket_permission: "0770"
 #   allowed_users:
 #     - alice@example.com
 #
+#   # Optional: Enable PKCE (Proof Key for Code Exchange) support for enhanced security
+#   # and prevent CSRF attacks.
+#   # See https://datatracker.ietf.org/doc/html/rfc7636
+#   enable_pkce: false
```

Comment:
If it is default off, I think we can leave it uncommented.

Are there any requirements to use this on the OIDC provider side? Should this just be on by default? what is the pro/con for having it as a default?

I think also this should be called:
```
pkce:
  enabled: false
```
to align with our config language.

Reply:
Good, the example needs to be updated